### PR TITLE
Fixes #23700 - passenger/http can manage symlinks

### PR DESCRIPTION
**Pull request by lzap**
Foreman Templates plugin needs this, ships with Satellite 6.3

### CODE REVIEW

File: foreman.te [hunk 1]
```diff
@@ -236,7 +239,11 @@ optional_policy(`
 ')
 
 tunable_policy(`httpd_run_foreman', `
-    allow httpd_t passenger_tmp_t:sock_file write;
+    manage_dirs_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
+    manage_files_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
+    manage_sock_files_pattern(httpd_t, passenger_tmp_t, passenger_tmp_t)
```
Comment (adamruzicka) [inline foreman.te:244]: I assume `manage_sock_files_pattern` replaces `allow httpd_t passenger_tmp_t:sock_file write;`, is that correct?

**Review by adamruzicka — APPROVED** [review]
Fixes the issue for me, left one comment inline just to check I'm interpreting the macros correctly.

File: foreman.te (hunk 1 above)
Comment (lzap) [reply foreman.te:244]: Exactly, see here: https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/support/file_patterns.spt#L373-L376

Grep the repo further to find `policy/support/obj_perm_sets.spt` file holding another macro with all the actions which includes the `write` operation. Just keep in mind when looking up macros that we target CentOS7 and git repo for that is internal (or use centos7 SRPM instead). There is lot of development already.

**Comment by lzap** [comment]
We have another report the patch works also with Satellite 6.3.

File: foreman.te (hunk 1 above)
Comment (lzap) [reply foreman.te:244]: To further explain, passenger_t domain is already allowed all above, but httpd_t is not. We want to keep both on par when the foreman_run_httpd flag is turned on. It was a bit behind.

**Comment by dgoetz** [comment]
Verified it successfully on Foreman 1.17.

**Comment by adamruzicka** [comment]
@ares (or anyone with the right permissions) Could you click the green button for us?

**Comment by lzap** [comment]
Thanks for your reviews, appreciated! @adamruzicka ++

**Pull request merged**